indico/modules/events/papers/controllers/base.py

```diff
@@ -95,4 +95,7 @@ def _check_paper_protection(self):
         Override this in case you want to check for more specific
         privileges than the generic "can access".
         """
-        return self.contribution.can_access(session.user)
+        return (self.contribution.is_user_associated(session.user, check_abstract=True) or
```

Comment:
wrong place.  check what inherits from this class... this code should go into the override in RHPaperTimeline